### ISSUE TITLE
dataset split

### ISSUE THREAD

**Issue by Ahri959** [linyq2117/SAMRefiner#7]
Hello, I am interested in your work. I would like to know how you guys divided the training set and test set on the DAVIS58 dataset when training the Iou pred head of SAMREFINER++.

**Comment by linyq2117** [comment]
Thanks for your interest. We didn't split the dataset into the training set and test set. During IoU adaptation, we didn't use the GT mask (only coarse mask) as supervision, so there is no need to split the dataset.

**Comment by linyq2117** [comment]
Closed for inactivity.
